activities/MazeWeb.activity/js/activity.js

```diff
@@ -1,6 +1,19 @@
 define(["sugar-web/activity/activity","tween","rAF","activity/directions","sugar-web/graphics/presencepalette", "sugar-web/env",  "sugar-web/graphics/icon", "webL10n", "sugar-web/graphics/palette", "rot", "humane"], function (activity, TWEEN, rAF, directions, presencepalette, env, icon, webL10n, palette, ROT, humane) {
 
-    requirejs(['domReady!'], function (doc) {
+    // Resize screen
+  document.getElementById("fullscreen-button").addEventListener('click', function() {
```

Comment:
Using classes/id for selectors on JS isnt a good pratice, instead use data-attributes. Classes and Ids main purpose is to stylize the html.

Reply:
But since this whole project is already using Id's I would suggest to stick with it and bring this conversation to the manteiners :+1: